extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -25,17 +25,21 @@ export const sqlProjectSdk = 'Microsoft.Build.Sql';
 
 // Project Provider
 export const emptySqlDatabaseProjectTypeId = 'EmptySqlDbProj';
-export const emptyProjectTypeDisplayName = localize('emptyProjectTypeDisplayName', "SQL Database");
-export const emptyProjectTypeDescription = localize('emptyProjectTypeDescription', "Develop and publish schemas for SQL databases starting from an empty project");
+export const emptyProjectTypeDisplayName = localize('emptyProjectTypeDisplayName', "SQL Server Database");
```

Comment:
should this be changed to "SQL Server"? I thought "SQL Server" implies on-prem, but Azure sql db and DW are also options for the target platform for this template

Reply:
@scoriani did we decide on SQL Server or SQL Server Database. I think we added "Database" because it's a database project and not project for server

Reply:
Generally speaking, if the artifact (e.g. SQL Proj) we're dealing with generates a "database", then i would say adding "Database" to the string makes sense. Different story would be if we would refer to a "target" for our artifacts, in that case it would be just "SQL Server". The exception i see here is "Azure SQL Database": because the product name here already contains the word "database", I wouldn't do "Azure SQL Database database"😊